packages/atomic/src/components/common/interface/i18n.ts

```diff
@@ -57,5 +58,8 @@ export function initi18n(atomicInterface: BaseAtomicInterface<AnyEngineType>) {
     nsSeparator: '___',
     fallbackLng: 'en',
     backend: i18nBackendOptions(atomicInterface),
+    interpolation: {
+      escape: (str) => DOMPurify.sanitize(str),
```

Comment:
Technically, with DOMPurify, we are now allowing HTML elements. I assume they weren't allowed before.

I don't really have an issue with that though.